tests/integration/views/test_session.py

```diff
@@ -9,4 +9,4 @@ def test_session_expired(self):
 
     def test_session_signed_out(self):
         self.get('/signed-out')
-        self.assertInPage('Your survey has been saved')
+        self.assertInPage('Your survey answers have been saved')
```

Comment:
Should we also add a test that checks the link goes to RRM/RAS prod/preprod?

Reply:
by asserting the href and not by clicking it